docs/deployment_patterns/how-to-instantiate-a-data-context-hosted-environments.md

```diff
@@ -46,7 +46,7 @@ In order to use an Expectation Suite you've created to validate data, follow thi
 Step 4: Use Data Docs
 ----------------------
 
-Finally, if you would like to build and view Data Docs in your environment, please follow the guides for configuring Data Docs: [Options for hosting Data Docs](../tutorials/getting-started/customize-your-deployment#options-for-hosting-data-docs).
+Finally, if you would like to build and view Data Docs in your environment, please follow the guides for configuring Data Docs: [Options for hosting Data Docs](../tutorials/getting_started/customize-your-deployment#options-for-hosting-data-docs).
```

Comment:
@cdkini GitHub still shows me file names containing hyphens, while the rest of the path uses the underscores.  Thanks.